src/components/ControlBar/ViewControlBar/DashboardsBar.js

```diff
@@ -0,0 +1,133 @@
+import React, { useState, useRef, useEffect, createRef } from 'react'
+import { connect } from 'react-redux'
+import cx from 'classnames'
+import PropTypes from 'prop-types'
+
+import Content from './Content'
+import ShowMoreButton from './ShowMoreButton'
+import DragHandle from './DragHandle'
+import { getRowsFromHeight } from './controlBarDimensions'
+import { sGetControlBarUserRows } from '../../../reducers/controlBar'
+import { acSetControlBarUserRows } from '../../../actions/controlBar'
+import { apiPostControlBarRows } from '../../../api/controlBar'
+
+import classes from './styles/DashboardsBar.module.css'
+
+export const MIN_ROW_COUNT = 1
+export const MAX_ROW_COUNT = 10
+const rowClassMap = {
+    [MIN_ROW_COUNT]: 'one',
```

Comment:
Shouldn't this be inverted, like this?

```
    const rowClassMap = {
        1 : 'one',
        2: 'two',
        3: 'three',
        4: 'four',
        5: 'five',
        6: 'six',
        7: 'seven',
        8: 'eight',
        9: 'nine',
        10: 'ten',
    }

    export const MIN_ROW_COUNT = Object.keys(rowClassMap)[0]
    export const MAX_ROW_COUNT = Object.keys(rowClassMap)[rowClassMap.length - 1]
```
Much more complicated, but seems more logically correct, right?
Or what happens if the min, max or the map changes otherwise?

Reply:
e.g. setting `MAX_ROW_COUNT = 11` would now result in the map containing `11: 'ten'` 🤷